pkg/ray-controller/k8s/apis/ray.io/v1/types.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// Ray is a specification for a Ray resource
+type Ray struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   RaySpec   `json:"spec"`
+	Status RayStatus `json:"status"`
+}
+
+// RaySpec is the spec for a Ray resource
+type RaySpec struct {
+	RayHead   RayHeadSpec   `json:"RayHeadSpec"`
+	RayWorker RayWorkerSpec `json:"RayWorkerSpec"`
+}
+
+// RayStatus is the status for a Ray resource
+type RayStatus struct {
+	Conditions []RayCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+}
+
+// RayHeadSpec is the spec for a Ray Head
+type RayHeadSpec struct {
+	Replicate int32 `json:"Replicate"`
```

Comment:
```suggestion
	Replicas int32 `json:"replicas"`
```

In K8s community, we prefer to use `Replicas`, I think. (https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L124 and https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L274)

Reply:
Thx for your suggestions.
I have fix it.